chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -244,11 +252,19 @@
   end
 
   users = {}
-  node["crowbar"]["users"].each do |k,h|
-    next if h["disabled"]
-    # Fix passwords into digests.
-    h["digest"] = Digest::MD5.hexdigest("#{k}:#{realm}:#{h["password"]}") if h["digest"].nil?
-    users[k] = h
+
+  begin
+    crowbarrc = IniFile.load("/etc/crowbarrc") || {}
+  rescue IniFile::Error
+    Chef::Log.warn("Could not parse config file /etc/crowbarrc")
```

Comment:
Any reason why this is not a fatal error?

Reply:
If this happens, we simply don't change the htdigest bits and let everything go. The reason this matters is that having chef-client fail on the admin server is pretty bad -- it would stop the discovery process from working for instance (since the dhcp config wouldn't get updated anymore).

Reply:
OK, so if you do followup PR, could you put this explanation into a comment?